sky/engine/core/dart/painting.dart

```diff
@@ -433,13 +482,29 @@ class ImageShader extends Shader {
 }
 
 /// Defines how a list of points is interpreted when drawing a set of triangles.
-/// See Skia or OpenGL documentation for more details.
 enum VertexMode {
+  /// Draw each sequence of three points as the vertices of a triangle.
   triangles,
+
+  /// Draw each sliding window of three points as the vertices of a triangle.
   triangleStrip,
+
+  /// Draw the first point and each sliding window of two points as the vertices of a triangle.
   triangleFan,
 }
 
+/// Defines how a list of points is interpreted when drawing a set of points.
+enum PointMode {
+  // Draw each point separately.
+  points,
+
+  // Draw each sequence of two points as a line segment.
+  lines,
+
+  // Draw the entire sequence of point as the vetices of a polygon.
+  polygon,
+}
+
```

Comment:
would be nice if VertexMode and PointMode discussed when you'd use each one
c.f. http://docs.flutter.io/flutter/dart-ui/BlurStyle-class.html